### PR TITLE
Documents

### PR DESCRIPTION
**Pull request by MathiasGagnepain**
- Adding Weekly report 2 content

### CODE REVIEW

**Review by EnzoGuillouche — APPROVED** [review]
QA's review: everything's good (already checked), ready to be pushed.

File: README.md
Comment (Guillaume18100) [inline README.md:1]: good

Comment (Guillaume18100) [inline README.md:1]: checked

File: documents/functional/AT2-Assembly-Language-Documentation.md
Comment (Guillaume18100) [inline documents/functional/AT2-Assembly-Language-Documentation.md:1]: checked

File: documents/functional/Functional Specification Document Template.md
Comment (Guillaume18100) [inline documents/functional/Functional Specification Document Template.md:1]: checked

File: documents/functional/functional.md
Comment (Guillaume18100) [inline documents/functional/functional.md:1]: checked

File: documents/management/management_artifact.md
Comment (Guillaume18100) [inline documents/management/management_artifact.md:1]: checked

File: documents/management/weekly_reports/cumulative.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/cumulative.md:1]: checked

File: documents/management/weekly_reports/week_1.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/week_1.md:1]: checked

File: documents/management/weekly_reports/week_2.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/week_2.md:1]: checked

File: documents/management/weekly_reports/week_3.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/week_3.md:1]: checked

File: documents/management/weekly_reports/week_4.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/week_4.md:1]: checked

File: documents/management/weekly_reports/week_5.md
Comment (Guillaume18100) [inline documents/management/weekly_reports/week_5.md:1]: checked

**Review by Guillaume18100 — APPROVED** [review]

**Pull request merged**